Modules/_lsprof.c

```diff
@@ -479,27 +489,23 @@ static PyStructSequence_Field profiler_subentry_fields[] = {
 
 static PyStructSequence_Desc profiler_entry_desc = {
     "_lsprof.profiler_entry", /* name */
-    NULL, /* doc */
+    "", /* doc */
```

Comment:
IMO PyType_FromSpec() should accept tp_doc=NULL, I created: https://bugs.python.org/issue41832

I don't ask you to change your PR, "" vs NULL can be changed again later.